numpy/core/src/private/npy_binsearch.h.src

```diff
@@ -100,10 +100,9 @@ static PyArray_@Arg@BinSearchFunc *gen@arg@binsearch_map[] = {
 static NPY_INLINE PyArray_@Arg@BinSearchFunc*
 get_@arg@binsearch_func(PyArray_Descr *dtype, NPY_SEARCHSIDE side)
 {
-    static npy_intp num_funcs = sizeof(_@arg@binsearch_map) /
-                                sizeof(_@arg@binsearch_map[0]);
+    npy_intp nfuncs = sizeof(_@arg@binsearch_map)/sizeof(@arg@binsearch_map);
```

Comment:
I think this was better using the `[0]`, because it ensure the types stay in sync.

Reply:
Hmm, in this case I don't think keeping things in sync will be a problem. However, the macro seems reasonable, just needs to be kept local to the header.

Reply:
Done.